Python/tests/test_runner_number.py

```diff
@@ -27,10 +27,12 @@ def test_number_recognizer(culture, model, options,
 
         assert_verbose(actual.type_name, expected['TypeName'], spec_info)
         assert_verbose(actual.text, expected['Text'], spec_info)
-        assert_verbose(actual.start, expected['Start'], spec_info)
-        assert_verbose(actual.end, expected['End'], spec_info)
         assert_verbose(actual.resolution['value'], expected['Resolution']['value'], spec_info)
 
+        if 'Start' in expected and 'End' in expected:
```

Comment:
Same as in Italian review.

Reply:
There were numerous German tests missing the `Start` and `End` keys so I had to add this check back in.